389-ds tests fail on Tumbleweed

Failure is here: https://github.com/SUSE/BCI-tests/actions/runs/8979806867/job/24662423700#step:11:2973

```ShellSession
FAILED tests/test_389ds.py::test_ldapwhoami[opensuse/389-ds:3.0 from registry.opensuse.org/devel/bci/tumbleweed/containerfile/opensuse/389-ds:3.0] - AssertionError: Unexpected exit code 1 for CommandResult(backend=<testinfra.backend.docker.DockerBackend object at 0x7fc310925b40>, exit_status=1, command=b'dsidm localhost initialise', _stdout=b'Enter basedn : ', _stderr=b'Error: EOF when reading a line\n')
```

Looks like `dsidm localhost initialise` needs a `basedn` and doesn't obtain it.

--- a/tests/test_389ds.py
+++ b/tests/test_389ds.py
@@ -8,37 +8,31 @@ CONTAINER_IMAGES = CONTAINER_389DS_CONTAINERS
 
 
 def test_ldapwhoami(auto_container_per_test):
-    auto_container_per_test.connection.run_expect(
-        [0],
-        "dsconf localhost backend create --suffix dc=example,dc=com --be-name userRoot",
-    )
-
-    auto_container_per_test.connection.run_expect(
-        [0], "dsidm localhost initialise"
+    basedn = "dc=suse,dc=com"
+    auto_container_per_test.connection.check_output(
+        f"dsconf localhost backend create --suffix {basedn} --be-name userroot --create-suffix --create-entries",
     )
 
     # Check a basic search
-    auto_container_per_test.connection.run_expect(
-        [0], "dsidm localhost account list"
+    auto_container_per_test.connection.check_output(
+        f"dsidm -b {basedn} localhost account list"
     )
 
     # set a dummy password on an account
-    auto_container_per_test.connection.run_expect(
-        [0],
-        "dsidm localhost account reset_password uid=demo_user,ou=people,dc=example,dc=com password",
+    auto_container_per_test.connection.check_output(
+        f"dsidm -b {basedn} localhost account reset_password uid=demo_user,ou=people,{basedn} password",
     )
 
     # Unlock the account
-    auto_container_per_test.connection.run_expect(
-        [0],
-        "dsidm localhost account unlock uid=demo_user,ou=people,dc=example,dc=com",
+    auto_container_per_test.connection.check_output(
+        f"dsidm -b {basedn} localhost account unlock uid=demo_user,ou=people,{basedn}",
     )
 
     host_port = auto_container_per_test.forwarded_ports[0].host_port
     if LOCALHOST.exists("ldapwhoami"):
         assert (
             LOCALHOST.check_output(
-                f"ldapwhoami -H ldap://127.0.0.1:{host_port} -x -D 'uid=demo_user,ou=people,dc=example,dc=com' -w password",
+                f"ldapwhoami -H ldap://127.0.0.1:{host_port} -x -D 'uid=demo_user,ou=people,{basedn}' -w password",
             )
-            == "dn: uid=demo_user,ou=people,dc=example,dc=com"
+            == f"dn: uid=demo_user,ou=people,{basedn}"
         )

